develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-theme-generator/01-installing-the-generator.markdown

```diff
@@ -0,0 +1,132 @@
+# Installing the Theme Generator [](id=installing-the-theme-generator)
+
+The Liferay Theme Generator has several dependencies. Installation varies 
+slightly between Windows and Unix-based systems. This tutorial covers how to 
+install the Liferay Theme Generator and its dependencies on both Windows and 
+Unix-based systems. 
+
+Start by installing the Theme Generator and its dependencies. 
+
+## Installing the Generator and Dependencies
+
+Follow these steps to install the Liferay Theme Generator:
+
+1.  Install [Node.js](http://nodejs.org/), if it's not already installed. To 
+    test your Node.js installation, execute the following command:
+
+        node -v
+
+    The resulting output should look similar to this:
+
+        v8.9.4
+
+    +$$$
+
+    Note: To avoid any potential compatibility issues, we recommend installing
+    the Long Term Support (LTS) version of Node.js--at the time of this writing,
+    the LTS version is v8.9.4.
+    
+    $$$
+    <!-- Is this still recommended? --> 
```

Comment:
@Robert-Frampton is the LTS version still recommended?

Reply:
Yeah LTS is still recommended.